hack/make-rules/test-integration.sh

```diff
@@ -90,8 +90,17 @@ if [[ -n "${KUBE_TEST_ARGS}" ]]; then
   runTests v1
 fi
 
+# Pass arguments that begin with "-" and move them to goflags.
```

Comment:
You may want `optparse` for parsing options ... not sure attempts to do things like this are a good idea, they can get bloated quickly and have rough edges.